aframe/tasks/train/train.py

```diff
@@ -82,7 +82,6 @@ def __init__(self, *args, **kwargs):
             raise ValueError(
                 "data_dir must be an s3 path for remote training tasks"
             )
-        authenticate()
```

Comment:
Why did the authentication location need to change?

Reply:
Was previously being called in the `__init__`, which is invoked at times inside certain sandboxes that don't have kubectl installed - so we just call it as needed now